tests/core/pyspec/eth2spec/test/helpers/epoch_processing.py

```diff
@@ -67,8 +92,17 @@ def run_epoch_processing_with(spec, state, process_name: str):
     Processes to the next epoch transition, up to and including the sub-transition named ``process_name``
       - pre-state ('pre'), state before calling ``process_name``
       - post-state ('post'), state after calling ``process_name``
+      - pre-full-state ('pre_full'), state before epoch transition
+      - post-full-state ('post_full'), state after epoch transition
+    The state passed by reference will be modified to be the ``process_name``post state.
```

Comment:
`*_full` naming makes it hard to re-read. Could you improve this, maybe `pre_epoch` and `post_epoch`?

Reply:
Done